components/kyma-environment-broker/internal/broker/plans_schema.go

```diff
@@ -58,13 +58,28 @@ type Type struct {
 	UniqueItems     *bool         `json:"uniqueItems,omitempty"`
 }
 
-func NameProperty() Type {
-	return Type{
-		Type:  "string",
-		Title: "Cluster Name",
-		// Allows for all alphanumeric characters, '_', and '-'
-		Pattern:   "^[a-zA-Z0-9-]*$",
-		MinLength: 1,
+type NameType struct {
+	Type
+	BTPdefaultTemplate BTPdefaultTemplate `json:"_BTPdefaultTemplate,omitempty"`
+}
+
+type BTPdefaultTemplate struct {
```

Comment:
nit: I am used to seeing a capital letter in camel case after an acronym, but I guess it's ok either way as long as we remain consistent across our own codebase :)